binding/rust/src/pvrecorder.rs

```diff
@@ -154,20 +149,18 @@ impl RecorderBuilder {
 
     pub fn init(&self) -> Result<Recorder, RecorderError> {
```

Comment:
In all other SDKs, we're calling the classes PvRecorder and PvRecorderError/Exception - can we follow that pattern here?